internal/apischema/openai/openai.go

```diff
@@ -431,7 +436,8 @@ type ChatCompletionUserMessageParam struct {
 	Role string `json:"role"`
 	// An optional name for the participant. Provides the model information to
 	// differentiate between participants of the same role.
-	Name string `json:"name,omitempty"`
+	Name  string      `json:"name,omitempty"`
+	Cache *CacheValue `json:"cache,omitempty"`
```

Comment:
Let's a comment for this field as it is anthropic specific

Reply:
fixed